app/models/service_template.rb

```diff
@@ -74,6 +77,7 @@ class ServiceTemplate < ApplicationRecord
   scope :without_service_template_catalog_id,       ->         { where(:service_template_catalog_id => nil) }
   scope :with_existent_service_template_catalog_id, ->         { where.not(:service_template_catalog_id => nil) }
   scope :displayed,                                 ->         { where(:display => true) }
+  scope :public_service_template,                   ->         { where(:internal => true) }
```

Comment:
Does this work?  I don't think you can use a `where` like this to query on a reserved attribute.

I tried it on another model and got:
```ruby
NoMethodError: undefined method `eq' for nil:NilClass
	from /home/bdunne/.gem/ruby/2.4.3/gems/activerecord-5.0.7/lib/active_record/relation/predicate_builder/basic_object_handler.rb:9:in `call'
	from /home/bdunne/.gem/ruby/2.4.3/gems/activerecord-5.0.7/lib/active_record/relation/predicate_builder.rb:76:in `build'
	from /home/bdunne/.gem/ruby/2.4.3/gems/activerecord-5.0.7/lib/active_record/relation/predicate_builder.rb:46:in `expand'
	from /home/bdunne/.gem/ruby/2.4.3/gems/activerecord-5.0.7/lib/active_record/relation/predicate_builder.rb:90:in `block in expand_from_hash'
	from /home/bdunne/.gem/ruby/2.4.3/gems/activerecord-5.0.7/lib/active_record/relation/predicate_builder.rb:86:in `each'
	from /home/bdunne/.gem/ruby/2.4.3/gems/activerecord-5.0.7/lib/active_record/relation/predicate_builder.rb:86:in `flat_map'
	from /home/bdunne/.gem/ruby/2.4.3/gems/activerecord-5.0.7/lib/active_record/relation/predicate_builder.rb:86:in `expand_from_hash'
	from /home/bdunne/.gem/ruby/2.4.3/gems/activerecord-5.0.7/lib/active_record/relation/predicate_builder.rb:31:in `build_from_hash'
	from /home/bdunne/.gem/ruby/2.4.3/gems/activerecord-5.0.7/lib/active_record/relation/where_clause_factory.rb:22:in `build'
	from /home/bdunne/.gem/ruby/2.4.3/gems/activerecord-5.0.7/lib/active_record/relation/query_methods.rb:632:in `where!'
	from /home/bdunne/.gem/ruby/2.4.3/gems/activerecord-5.0.7/lib/active_record/relation/query_methods.rb:625:in `where'
	from /home/bdunne/.gem/ruby/2.4.3/gems/activerecord-5.0.7/lib/active_record/querying.rb:10:in `where'
	from (irb):2
	from /home/bdunne/.gem/ruby/2.4.3/gems/railties-5.0.7/lib/rails/commands/console.rb:65:in `start'
	from /home/bdunne/.gem/ruby/2.4.3/gems/railties-5.0.7/lib/rails/commands/console_helper.rb:9:in `start'
	from /home/bdunne/.gem/ruby/2.4.3/gems/railties-5.0.7/lib/rails/commands/commands_tasks.rb:78:in `console'
	from /home/bdunne/.gem/ruby/2.4.3/gems/railties-5.0.7/lib/rails/commands/commands_tasks.rb:49:in `run_command!'
	from /home/bdunne/.gem/ruby/2.4.3/gems/railties-5.0.7/lib/rails/commands.rb:18:in `<top (required)>'
	from bin/rails:4:in `require'
	from bin/rails:4:in `<main>'
```